assets/js/helpers/boot.js

```diff
@@ -74,6 +74,24 @@ $(document).ready(function() {
           }
     });
   });
+  $.getScript("https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.js",function(){
+    window.bootTopic();
+    $.getScript("https://cdn.mathjax.org/mathjax/latest/MathJax.js",function(){
+      MathJax.Hub.Config({
+          jax: ["input/TeX","input/MathML","input/AsciiMath","output/CommonHTML"],
+          extensions: ["tex2jax.js","mml2jax.js","asciimath2jax.js","MathMenu.js","MathZoom.js","AssistiveMML.js"],
+          TeX: {
+            extensions: ["AMSmath.js","AMSsymbols.js","noErrors.js","noUndefined.js"]
+          },
+          tex2jax: {
+            skipTags:["code"]
+          }
+        });
+      MathJax.Hub.Queue(["Typeset",MathJax.Hub]);
+      $.getScript("/js/dependencies/async_loader.js",function(){
```

Comment:
The thing is that everything execute after document.ready event, so all script should be available normally.